### DISCUSSION TITLE
Welcome to ernest-phillips.github.io Discussions!

### DISCUSSION THREAD

**Discussion by ernest-phillips** [ernest-phillips/Engineer-in-training#7] [General]
## 👋 Welcome! 

  I was able to get Discussions access for this repo.
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by looshlee** [comment]
I love this idea!

**Comment by Mattamorphic** [comment]
Hey folks! I'm Matt, Software Engineer for Support @ GitHub (incase there are folks outside of GitHub collaborating here!) 

Happy to help with any engineering questions, from design, and development to theory and project management skills ✨
